srsue/src/stack/rrc_nr/rrc_nr.cc

```diff
@@ -679,6 +682,17 @@ void rrc_nr::send_ul_dcch_msg(uint32_t lcid, const ul_dcch_msg_s& msg)
     log_rrc_message(get_rb_name(lcid), Tx, pdu.get(), msg, msg.msg.c1().type().to_string());
   }
 
+  // FUZZING HERE
+  if(args.sdu_fuzzed_bits > 0){
```

Comment:
Odd spacing on the if statement here. We might want to use a tool like clang-tidy and/or clang-format to enforce code style. @joshuamoorexyz, any thoughts?

Reply:
I made some edits to the spacing, let me know what you think. I am still learning about code quality, spacing etc.